src/app/components/OrderBook.tsx

```diff
@@ -109,31 +113,36 @@ export function OrderBook() {
   const buys = useAppSelector((state) => state.orderBook.buys);
 
   return (
-    <div className="p-2 text-sx">
-      <div className="sized-columns max-w-sm">
-        <div className="">
-          Order
-          <br />
-          Count
-        </div>
-        <div className="text-end">
-          Price
-          <br />({token2Symbol})
-        </div>
-        <div className="text-end">
-          Size
-          <br />({token1Symbol})
+    <div className="p-2 text-sx text-primary-content">
+      <div className="sized-columns">
+        <div className="mx-2 my-4 whitespace-nowrap text-secondary-content">
```

Comment:
@EvgeniiaVak we do not need Order Book now, it was just placed by me for understanding the placement of the layout component

Reply:
I took it from Vlad's figma. Also, I think I like it, but we can remove of course. Did you try with and without it, what looks better to you?

Reply:
ok, removing, it's ok without it too. Oh, also I forgot the narrow screen fix, back to draft for now.

Reply:
i think the order book component is pretty much self explanatory to the users. and besides we are not giving headings to other components. so i would suggest to keep it consistent with other components

Reply:
> i think the order book component is pretty much self explanatory to the users. and besides we are not giving headings to other components. so i would suggest to keep it consistent with other components

referring to this figma design 
![image](https://github.com/DeXter-on-Radix/website/assets/29409407/38c8bfaf-07d2-477c-8a67-06a13a5275fa)